JUnit version 4.13.2
......................................................................................................................E............................................................................................................................E.................................
Time: 22.991
There were 2 failures:
1) test055(org.flywaydb.core.api.configuration.ClassicConfiguration_ESTest)
java.lang.AssertionError: Exception was not thrown in org.flywaydb.core.internal.logging.buffered.BufferedLog but in org.flywaydb.core.internal.logging.EvolvingLog.debug(EvolvingLog.java:52): java.lang.NullPointerException
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:111)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:48)
	at org.flywaydb.core.api.configuration.ClassicConfiguration_ESTest.test055(ClassicConfiguration_ESTest.java:651)
2) test059(org.flywaydb.core.api.configuration.ClassicConfiguration_ESTest)
java.lang.AssertionError: Exception was not thrown in org.flywaydb.core.internal.logging.buffered.BufferedLog but in org.flywaydb.core.internal.logging.EvolvingLog.debug(EvolvingLog.java:52): java.lang.NullPointerException
	at org.evosuite.runtime.EvoAssertions.assertThrownBy(EvoAssertions.java:111)
	at org.evosuite.runtime.EvoAssertions.verifyException(EvoAssertions.java:48)
	at org.flywaydb.core.api.configuration.ClassicConfiguration_ESTest.test059(ClassicConfiguration_ESTest.java:720)

FAILURES!!!
Tests run: 275,  Failures: 2

